examples/flower-authentication/README.md

```diff
@@ -82,35 +78,51 @@ flower-superlink \
     --auth-superlink-public-key keys/server_credentials.pub
 ```
 
-## Start the long-running Flower client (SuperNode)
+Then launch the `SuperExec`:
+
+```bash
+flower-superexec \
+    --ssl-ca-certfile certificates/ca.crt \
+    --ssl-certfile certificates/server.pem \
+    --ssl-keyfile certificates/server.key \
+    --executor-config '--executor-config 'root-certificates=\"certificates/ca.crt\"'' \
+    --executor flwr.superexec.deployment:executor
+
+```
+
+At this point your server-side is idling. First, let's connect two `SuperNodes`, and then we'll start a run.
+
+## Start the long-running Flower client-side (SuperNode)
 
 In a new terminal window, start the first long-running Flower client (SuperNode):
 
 ```bash
-flower-client-app client:app \
+flower-supernode \
     --root-certificates certificates/ca.crt \
-    --server 127.0.0.1:9092 \
+    --superlink 127.0.0.1:9092 \
     --auth-supernode-private-key keys/client_credentials_1 \
-    --auth-supernode-public-key keys/client_credentials_1.pub
+    --auth-supernode-public-key keys/client_credentials_1.pub \
+    --node-config "partition-id=0 num-partitions=2"
```

Comment:
Hi @danielnugraha , this README looks pretty good already. I made some small changes to align it better with other readmes. I wonder if we should not make use of `partition-id` and `num-partitions` since that's something people would rarely (never?) use in a deployment setup. I'm inclined to remove this `--node-config` and hardcode a partition in the `client_fn()` and leave a comment there saying "here load data in your SuperNode host". wdyt? @danieljanes, what's your view?

Reply:
thanks @danielnugraha for starting this effort!